config/namelist.io

```diff
@@ -1,46 +1,65 @@
+ldiag_curl_vel3   = .false.  ! enables 'curl_u' output (relative vorticity from 3D velocity)
+ldiag_Ri          = .false.  ! enables Richardson number diagnostics ('shear', 'Ri')
+ldiag_turbflux    = .false.  ! enables turbulent flux diagnostics ('KvdTdz', 'KvdSdz')
+ldiag_salt3D      = .false.  ! enables 3D salinity diagnostics
+ldiag_dMOC        = .false.  ! enables 'dMOC' output (density MOC diagnostics)
+ldiag_DVD         = .false.  ! enables 'DVD' output (Discrete Variance Decay diagnostics)
+ldiag_forc        = .false.  ! enables 'FORC' output (comprehensive forcing diagnostics)
+ldiag_extflds     = .false.  ! enables extended field diagnostics
+ldiag_destine     = .false.  ! enables heat content computation ('hc300m', 'hc700m', 'hc')
+ldiag_trflx       = .false.  ! enables tracer flux diagnostics ('utemp', 'vtemp', 'usalt', 'vsalt')
+ldiag_uvw_sqr     = .false.  ! enables 'UVW_SQR' output (squared velocities: u2, v2, w2)
+ldiag_trgrd_xyz   = .false.  ! enables 'TRGRD_XYZ' output (horizontal & vertical tracer gradients)
 /
 
+! ============================================================================
+! GENERAL OUTPUT SETTINGS
+! ============================================================================
 &nml_general
-io_listsize    =120 !number of streams to allocate. shallbe large or equal to the number of streams in &nml_list
-vec_autorotate =.false.
-compression_level = 1
+io_listsize       = 120      ! total number of streams to allocate. Shall be larger or equal to the number of streams in &nml_list (max. 150)
+vec_autorotate    = .false.  ! unrotate vector fields (velocities, winds) before writing to output files
+compression_level = 1        ! compression level for netCDF output (1=fastest, 9=smallest)
 /
 
-! for sea ice related variables use_ice should be true, otherewise there will be no output
-! for 'curl_surf' to work lcurt_stress_surf must be .true. otherwise no output
-! for 'fer_C', 'bolus_u', 'bolus_v', 'bolus_w', 'fer_K' to work Fer_GM must be .true. otherwise no output
-! 'otracers' - all other tracers if applicable
-! for 'dMOC' to work ldiag_dMOC must be .true. otherwise no output
-! for 'utemp', 'vtemp', 'usalt', 'vsalt' output, set ldiag_trflx=.true.
+! ============================================================================
+! OUTPUT VARIABLE LIST
+! ============================================================================
+! Format: 'variable_id', frequency, unit, precision
+!   frequency  = output frequency (integer)
+!   unit       = 'y' (yearly), 'm' (monthly), 'd' (daily), 'h' (hourly), 's' (steps)
+!   precision  = 4 (single precision) or 8 (double precision)
+! ============================================================================
 &nml_list
 io_list =  'sst       ',1, 'm', 4,
            'sss       ',1, 'm', 4,
-    	   'ssh       ',1, 'm', 4,
-           'uice      ',1, 'm', 4,
-           'vice      ',1, 'm', 4,
+           'ssh       ',1, 'm', 4,
+           'uice      ',1, 'd', 4,
```

Comment:
Why change to `d` for uice and vice? I would say by default we should have months everywhere.

Reply:
Good point, I changed all default output frequencies to monthly, in both the standard output list and the catalogue below.